File: hw/aimacana/u3/hw25CompositePattern/java/Composite/src/main/java/ec/edu/espe/composite/model/Teller.java
package ec.edu.espe.composite.model;

public class Teller extends Employee {
    public Teller() {
        title = "Teller";
    }

    public Teller(String aName) {
        this();
        name = aName;
    }
    
    @Override
    public String stateName() {
        return super.stateName();
    }
}

File: hw/collahuazo/u3/hw24CompositePaterns/CompositePatternJS/composite/model/Teller.js
const { Employee } = require("./Employee.js");

class Teller extends Employee {
  constructor(aName) {
    super();
    this.title = "Teller";
    this.name = aName;
  }

  stateName() {
    super.stateName();
  }
}

module.exports = { Teller };
